deploy/components/inference-gateway/deployments.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       terminationGracePeriodSeconds: 130
       containers:
       - name: epp
-        image: ghcr.io/llm-d/llm-d-inference-scheduler:latest
+        image: ${EPP_IMAGE}
```

Comment:
Please undo this change. The file as it was allows the YAML file to be used outside of the kind based tests.